bodo/src/test/java/org/apache/calcite/test/BodoSqlToRelConverterTest.java

```diff
@@ -99,4 +111,62 @@ public class BodoSqlToRelConverterTest extends SqlToRelTestBase {
         "on value.value = dept2.value";
     sql(sql).ok();
   }
+
+  @Test void testCreateTableOrderBy() {
+    // Tests an error specific to CREATE TABLE with "WITH" and "ORDER BY" clauses during
+    // SqlToRelConversion. The converter was previously optimizing out the sort node,
```

Comment:
Are we still able to optimize out the sort?

Reply:
It looks like no. We can probably handle this as an optimization rule in general.